cosmwasm/submit-proposal.js

```diff
@@ -161,6 +161,41 @@ const execute = async (client, wallet, config, options) => {
     await callSubmitProposal(client, wallet, config, options, proposal);
 };
 
+const registerItsChain = async (client, wallet, config, options) => {
+    const chains = options.chains.map((chain) => {
+        const chainConfig = getChainConfig(config, chain);
+
+        if (!chainConfig.maxUintBits) {
```

Comment:
use `validateParameters` util, see existing usage. it'll help simplify validation everywhere in cosmwasm scripts